lib/rubocop/config_loader.rb

```diff
@@ -15,6 +15,7 @@ class ConfigNotFoundError < Error
   # directories are inspected.
   class ConfigLoader
     DOTFILE = '.rubocop.yml'.freeze
+    CONFIGFILE = 'rubocop/config.yml'.freeze
```

Comment:
Probably this should be `.rubocop`, not `rubocop`. Usually config dirs are "hidden". I'd also name the constant `CONFIG_FILE`.

Reply:
Files in XDG_CONFIG_HOME should not be hidden according to specs. The most common convention is `appname/config`.